ga4gh/datamodel/variants.py

```diff
@@ -547,3 +589,160 @@ def buildMetadata(
                         number="{}".format(value.number),
                         description=description))
         return ret
+
+
+class HtslibVariantAnnotationSet(HtslibVariantSet):
+    """
+    Class representing a single variant annotation set backed by a directory of
+    indexed VCF or BCF files.
+    """
+    def __init__(self, parentContainer, localId, dataDir, backend):
+        super(HtslibVariantAnnotationSet, self).__init__(
+            parentContainer, localId, dataDir, backend)
+        self.compoundIdClass = datamodel.VariantAnnotationSetCompoundId
+        self._compoundId = datamodel.VariantAnnotationSetCompoundId(
+            self.getCompoundId(), 'variantannotations')
+        self._sequenceOntology = backend.getOntology('sequence_ontology')
+
+    def getVariantAnnotations(self, referenceName, startPosition, endPosition):
```

Comment:
I'm having trouble getting search by feature ID to work. The schema suggests we should be able to find them by either reference, start, and stop, or a list of feature IDs. When I send in a request with a featureId array the server doesn't return any results.